tabular/src/autogluon/tabular/models/xgboost/xgboost_model.py

```diff
@@ -132,6 +131,7 @@ def _fit(self,
             callbacks.append(EarlyStoppingCustom(early_stopping_rounds, start_time=start_time, time_limit=time_limit, verbose=verbose))
             params['callbacks'] = callbacks
 
+        import xgboost
```

Comment:
Why is this import added? It shouldn't be necessary.

Reply:
debug code forgot to remove :(